api_testrail.py

```diff
@@ -331,6 +386,20 @@ def test_suites_delete(self):
         self.session.query(TestSuites).delete()
         self.session.commit()
 
+    def test_plans_delete(self):
```

Comment:
it is nice to have a method to delete each table, it is clear but creates duplication. I know we have already one specific for test_suite_delete, but it may be better to have just one where we pass the table name to delete... I have been doing that in recent implementations:
https://github.com/mozilla-mobile/testops-dashboard/blob/9a3918b257212d2528281d48f719bdeedfceb0fe/api_bugzilla.py#L266